Update for GitKraken v8.3.3

Anselmoo opened the pull request:
It seems to be that a few new variables were added 

![image](https://user-images.githubusercontent.com/13209783/160679581-8e06df91-646f-4624-8b77-e451b8badec6.png)

* Update `dracula-theme.jsonc`
* Add new screenshot

> If you're fixing a UI issue, make sure you take two screenshots. One that shows the actual bug and another that shows how you fixed it.

Xtopher98 commented on dracula-theme.jsonc:
```diff
@@ -6,35 +6,49 @@
-      "cyan": "#8be9fd",
-      "app__bg0": "#282a36",
-      "comment": "#6272a4",
+      "blue": "#bd93f9",
```
Why change purple to blue? The color in the Dracula documentation is called purple.

Xtopher98 commented on dracula-theme.jsonc:
```diff
@@ -6,35 +6,49 @@
-      "app__bg0": "#282a36",
-      "comment": "#6272a4",
+      "blue": "#bd93f9",
+      "ltblue": "#8be9fd",
```
Same with this. We should keep the same naming convention that Dracula does in [this list](https://draculatheme.com/contribute). This should stay cyan
```suggestion
      "cyan": "#8be9fd",
```

Xtopher98 commented on dracula-theme.jsonc:
```diff
@@ -6,35 +6,49 @@
+      "section-border": "fade(@foreground, 8%)",
+      "subtle-border": "fade(@foreground, 4%)",
+      "modal-overlay-color": "@background",
+      // graph colors
```
Why change the order of the graph colors? We should keep it consistent unless there is a good reason for the change. I do like the fade for the later colors.
```suggestion
      "graph-color-0": "@purple",
      "graph-color-1": "@cyan",
      "graph-color-2": "@green",
      "graph-color-3": "@orange",
      "graph-color-4": "@yellow",
      "graph-color-5": "fade(@purple, 8%)",
      "graph-color-6": "fade(@cyan, 8%)",
      "graph-color-7": "fade(@green, 8%)",
      "graph-color-8": "fade(@orange, 8%)",
      "graph-color-9": "fade(@yellow, 8%)",
```

Xtopher98 commented on dracula-theme.jsonc:
```diff
@@ -6,35 +6,49 @@
       "panel__bg1": "lighten(@panel__bg0, 5%)",
       "panel__bg2": "lighten(@panel__bg1, 5%)",
-      "input__bg": "fade(@app__bg0, 20%)",
+      "input__bg": "fade(@background, 10%)",
```
Why the fade percent change? Unless there is a good reason we should keep it consistent.
```suggestion
      "input__bg": "fade(@background, 20%)",
```

Xtopher98 commented on dracula-theme.jsonc:
```diff
@@ -6,35 +6,49 @@
       "text-secondary": "fade(@.text-color, 60%)",
       "text-disabled": "fade(@.text-color, 40%)",
-      "text-accent": "@purple",
+      "text-accent": "@blue",
```
Same as above
```suggestion
      "text-accent": "@purple",
```

Xtopher98 commented on dracula-theme.jsonc:
```diff
@@ -43,9 +57,9 @@
-      "primary-border": "@purple",
-      "primary-bg": "fade(@purple, 20%)",
-      "primary-hover": "fade(@purple, 60%)",
+      "primary-border": "@blue",
+      "primary-bg": "fade(@blue, 20%)",
+      "primary-hover": "fade(@blue, 60%)",
```
Same as above 
```suggestion
      "primary-border": "@purple",
      "primary-bg": "fade(@purple, 20%)",
      "primary-hover": "fade(@purple, 60%)",
```

Xtopher98 commented on dracula-theme.jsonc:
```diff
@@ -6,35 +6,49 @@
+      "subtle-border": "fade(@foreground, 4%)",
+      "modal-overlay-color": "@background",
+      // graph colors
+      "graph-color-0": "@ltblue",
+      "graph-color-1": "@pink",
+      "graph-color-2": "@blue",
+      "graph-color-3": "@orange",
+      "graph-color-4": "@comment",
+      "graph-color-5": "fade(@ltblue, 8%)",
+      "graph-color-6": "fade(@pink, 8%)",
+      "graph-color-7": "fade(@blue, 8%)",
+      "graph-color-8": "fade(@orange, 8%)",
+      "graph-color-9": "fade(@comment, 8%)",
```
Why change the order of the graph colors? We should keep it consistent unless there is a good reason for the change. I do like the fade for the later colors.
```suggestion
      "graph-color-0": "@purple",
      "graph-color-1": "@cyan",
      "graph-color-2": "@green",
      "graph-color-3": "@orange",
      "graph-color-4": "@yellow",
      "graph-color-5": "fade(@purple, 8%)",
      "graph-color-6": "fade(@cyan, 8%)",
      "graph-color-7": "fade(@green, 8%)",
      "graph-color-8": "fade(@orange, 8%)",
      "graph-color-9": "fade(@yellow, 8%)",
```

Xtopher98 commented on dracula-theme.jsonc:
```diff
@@ -56,53 +70,108 @@
-      "link-color": "@cyan",
-      "form-control-focus": "@purple",
+      "input--disabled": "fade(@background, 10%)",
+      "link-color": "@ltblue",
+      "form-control-focus": "@blue",
```
```suggestion
      "link-color": "@cyan",
      "form-control-focus": "@purple",
```

Xtopher98 commented on dracula-theme.jsonc:
```diff
@@ -56,53 +70,108 @@
       "scroll-thumb-bg": "@toolbar__bg2",
       "scroll-thumb-bg-light": "lighten(@toolbar__bg2, 20%)",
-      "wip-status": "fade(@green, 40%)",
+      "wip-status": "fade(@blue, 20%)",
```
```suggestion
      "wip-status": "fade(@purple, 20%)",
```

Xtopher98 commented on dracula-theme.jsonc:
```diff
@@ -56,53 +70,108 @@
+      "label__purple-color": "@blue",
       // component states
-      "filtering": "fade(@purple, 50%)",
+      "filtering": "fade(@blue, 50%)",
```
```suggestion
      "filtering": "fade(@purple, 50%)",
```

Xtopher98 commented on dracula-theme.jsonc:
```diff
@@ -56,53 +70,108 @@
+      "filtering": "fade(@blue, 50%)",
       "soloing": "fade(@orange, 50%)",
-      "checked-out": "fade(@purple, 30%)",
+      "checked-out": "fade(@green, 30%)",
```
Why change the color?

Xtopher98 reviewed (changes requested):
Good update overall adding the new values. 

We should keep the color names consistent with the [dracula documentation](https://draculatheme.com/contribute). I've added some suggestions to revert back.

I've added some other comments and suggestions in various places that I think are worth taking a look at. 

Let me know what you think.

Anselmoo commented:
> Good update overall adding the new values.
> 
> We should keep the color names consistent with the [dracula documentation](https://draculatheme.com/contribute). I've added some suggestions to revert back.
> 
> I've added some other comments and suggestions in various places that I think are worth taking a look at.
> 
> Let me know what you think.

Thx a lot @Xtopher98, the suggestion makes sense and you checked the PR very carefully 👍

I will later make a new screen-shot and finish the PR
